SU-HSDO/suhumsci
# [HOTFIX] STN-993: Vertical Timeline Utility Class Fix

# READY FOR REVIEW

## Summary
Incorrect box sizing was causing the circle and square icons, whose CSS was changed in the original STN-965 branch, to display incorrectly. The error was noticed in Firefox on all timeline items and in Chrome when a timeline item had no descriptive text, in which cases the circles and squares appeared to be about 50% of the size they were intended.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. View a page with a vertical timeline (suggested node: /node/42821)
2. Add a timeline item that has no descriptive body text

- [x] Confirm that the circles/squares of each timeline item are uniform in size and centered on the vertical line which connects the timeline items. 
- [x] Check the display in Firefox and Safari to confirm they appear the same as in Chrome

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
